### PR TITLE
[gtl] Update version to 1.1.8

### PR DESCRIPTION
**Pull request by greg7mdp**
- [ x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [x] The "supports" clause reflects platforms that may be fixed by this new version
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/gtl/portfile.cmake [hunk 1]
```diff
@@ -3,8 +3,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO greg7mdp/gtl
-    REF v1.1.5
-    SHA512 87666a975d3e355d24463f31bd7fe9796eb9e90b557b678807432fc3797a2b501e76acefae490cbaa503f993efe7f44c04a7cc8bd48662c38293c90741c8f7df
+    REF v1.1.6
```
Comment (jimwang118) [inline ports/gtl/portfile.cmake:6]:
```suggestion
    REF "v${VERSION}"
```

Comment (jimwang118) [reply ports/gtl/portfile.cmake:6]: I just checked that the latest upstream version is updated to v1.1.7, should we update to the latest version?

**Review by jimwang118 — CHANGES_REQUESTED** [review]

File: ports/gtl/portfile.cmake (hunk 1 above)
Comment (greg7mdp) [reply ports/gtl/portfile.cmake]: I just updated the latest release to v1.1.8, and this PR as well.

File: ports/gtl/portfile.cmake [hunk 2]
```diff
@@ -3,8 +3,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO greg7mdp/gtl
-    REF v1.1.5
-    SHA512 87666a975d3e355d24463f31bd7fe9796eb9e90b557b678807432fc3797a2b501e76acefae490cbaa503f993efe7f44c04a7cc8bd48662c38293c90741c8f7df
+    REF v1.1.8
```
Comment (jimwang118) [inline ports/gtl/portfile.cmake:6]:
```suggestion
    REF "v${VERSION}"
```

**Review by jimwang118 — CHANGES_REQUESTED** [review]

**Review by jimwang118 — DISMISSED** [review]

**Comment by jimwang118** [comment]
The usage has been tested successfully locally.

**Review by BillyONeal — APPROVED** [review]
Thanks for the update!

**Pull request merged**